packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -365,7 +365,7 @@ describe( 'Basic rendering', () => {
 				/>
 			);
 
-			const linkPreview = screen.getByLabelText( 'Currently selected' );
+			const linkPreview = screen.getByLabelText( 'Manage link' );
```

Comment:
We should prefer the `getByRole` selector.  This is [documented in the guide for Playwright](https://github.com/WordPress/gutenberg/blob/73a17b0d3dc9e08aaeb84bf03c9398d61a327a8c/docs/contributors/code/e2e/README.md#use-accessible-selectors) and we should aim to abide by this when writing tests.
```suggestion
			const linkPreview = screen.getByRole( 'group', {
				name: 'Manage link',
			} );
```

There are other occurrences within this file that also need fixing to utilise the new `role`s you've added.

Reply:
I'd agree usint the role selector is better, but this is already existing code, I was just changing the label value.